Mendelugram/Mendelugram/View/Photo/PhotoTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PhotoTableViewCell.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class PhotoTableViewCell: UITableViewCell {
+
+    static let cellHeight: CGFloat = 430
+    static let reuseIdentifier = String(describing: PhotoTableViewCell.self)
```

Comment:
If you want to have it like this for sake of simplicity then no problem but it would be nicer if you defined this in an UITableViewCell extension or a protocol with protocol extenstion